### PR TITLE
Splitting Gulp file into multiple files

### PR DESCRIPTION
**Pull request by seralaci**
Each gulp task has been broken out into its own file in gulp/tasks.

### CODE REVIEW

File: gulp/config.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+var path          = require('path');
+var environments  = require('gulp-environments');
+
+module.exports = {
+    src     : 'src/AppBundle/Resources/assets',
+    dest    : 'src/AppBundle/Resources/public',
+    dev     : environments.development,
+    prod    : environments.production
+};
```
Comment (sagikazarmark) [inline gulp/config.js]: New lines are missing from files. Git prefers having an empty new line at the end of each file.

**Comment by sagikazarmark** [comment]
Nice one Laci :+1: 

Added one small issue related to git. I only added it to one file, but as I can see, every file is affected by the issue.

Try installing editorconfig plugin in your IDE which will make sure that no file is saved without the trailing new line.

**Comment by seralaci** [comment]
Added the missing trailing new lines, see 64807f1

**Comment by sagikazarmark** [comment]
Awesome :+1: 
Thanks for the contribution.

**Pull request merged**